windows-driver-docs-pr/usbcon/usb-registry-settings-for-a-function-controller-driver.md

```diff
@@ -51,48 +51,46 @@ This table describes the subkeys that OEMs can modify under this key. More infor
-|--------------------------|------------|-------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| **idVendor**             | REG\_DWORD | OEM   | The vendor identifier for the device descriptor that is sent to the host during enumeration.                                                                                                                                                                                                                               |
-| **idProduct**            | REG\_DWORD | OEM   | The product identifier for the device descriptor that is sent to the host during enumeration.                                                                                                                                                                                                                              |
-| **ManufacturerString**   | REG\_SZ    | OEM   | The manufacturer string that is sent to the host to identify the manufacturer of the device.                                                                                                                                                                                                                               |
-| **ProductString**        | REG\_SZ    | OEM   | A string that describes the device as a product. The default value is Windows 10 Mobile Device. This value is used as the display name of the device in the connected computer's user interface. OEMs should make sure that this value matches the value of the PhoneModelName value under the DeviceTargetingInfo subkey. |
-| **iSerialNumber**        | REG\_DWORD | OEM   | If this value is set to 0, then the device does not have a serial number. If this value is non-zero or does not exist, then the serial number is generated uniquely per device.                                                                                                                                            |
-| **CurrentConfiguration** | REG\_SZ    | OEM   | This string must correspond to the name of a configuration subkey. This string determines which configuration to use to build a configuration descriptor for USB device enumeration.                                                                                                                                       |
+| Value | Type | Owner | Description|
+|--|--|--|--|
+| **IncludeDefaultCfg** | REG\_DWORD | OEM | Set to 0 when OEMs do not want to include the interfaces of the Default configuration such as IpOverUsb or MTP. When ommited, the default value is 1. |
```

Comment:
the statement for the omitted case might be incorrect. I will double-check and update the PR.

Reply:
updated in commit #2